### PR TITLE
Fixture composition

### PR DESCRIPTION
**Pull request by aharonha**

### CODE REVIEW

**Comment by aharonha** [comment]
@segabriel please review conceptually

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/CompositeFixture.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package io.scalecube.test.fixtures;
+
+import java.util.List;
+import java.util.stream.Stream;
+import org.opentest4j.AssertionFailedError;
+import org.opentest4j.TestAbortedException;
+
+final class CompositeFixture implements Fixture {
+  private final Fixture base;
+  private final List<Fixture> paramters;
```
Comment (segabriel) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/CompositeFixture.java]: Please fix typo

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/FixturesExtention.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package io.scalecube.test.fixtures;
+
+import java.util.function.Function;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+interface FixturesExtention {
```
Comment (segabriel) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/FixturesExtention.java]: Please fix typo --> `extension`

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/CompositeFixture.java (hunk 1 above)
Comment (aharonha) [reply scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/CompositeFixture.java]: 97bfac14884f6c84d6cc2aa68f2bfb1f6a7c29ef

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/FixturesExtention.java (hunk 2 above)
Comment (aharonha) [reply scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/FixturesExtention.java]: 97bfac14884f6c84d6cc2aa68f2bfb1f6a7c29ef

**Pull request merged**